### PR TITLE
Support compiling for Windows with OpenWatcom

### PR DESCRIPTION
**Pull request by ccawley2011**

### CODE REVIEW

File: Makefile.nt [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Makefile for Windows NT using Watcom compiler.
+#
+# wmake -f Makefile.nt
+
+CC=wcc386
+CPPFLAGS=-DHAVE_SIGNAL_H -DHAVE_SYS_TIME_H -DWIN32
```
Comment (sezero) [inline Makefile.nt:6]: No sys/time.h in ow19, and it builds fine without sys/time.h: please remove -DHAVE_SYS_TIME_H flag.

**Review by sezero — CHANGES_REQUESTED** [review]

File: Makefile.nt (hunk 1 above)
Comment (ccawley2011) [reply Makefile.nt]: OK, done.

**Review by sezero — APPROVED** [review]
Looks OK to me.  Will apply after libxmp side is applied.

**Comment by sezero** [comment]
One minor issue: to link statically to libxmp, -DBUILDING_STATIC is needed in CFLAGS (OK, expected that.)
With that though, linkage succeeds but with warnings:
```
Warning! W1027: file xmp_static.lib(load.c): redefinition of xmp_test_module ignored
Warning! W1027: file xmp_static.lib(load.c): redefinition of xmp_test_module_from_memory ignored
Warning! W1027: file xmp_static.lib(load.c): redefinition of xmp_test_module_from_file ignored
Warning! W1027: file xmp_static.lib(load.c): redefinition of xmp_load_module ignored
Warning! W1027: file xmp_static.lib(load.c): redefinition of xmp_load_module_from_memory ignored
Warning! W1027: file xmp_static.lib(load.c): redefinition of xmp_load_module_from_file ignored
Warning! W1027: file xmp_static.lib(load.c): redefinition of xmp_release_module ignored
Warning! W1027: file xmp_static.lib(load.c): redefinition of xmp_scan_module ignored
Warning! W1027: file xmp_static.lib(player.c): redefinition of libxmp_player_set_release ignored
Warning! W1027: file xmp_static.lib(player.c): redefinition of libxmp_player_set_fadeout ignored
Warning! W1027: file xmp_static.lib(player.c): redefinition of xmp_start_player ignored
Warning! W1027: file xmp_static.lib(player.c): redefinition of xmp_play_frame ignored
Warning! W1027: file xmp_static.lib(player.c): redefinition of xmp_play_buffer ignored
Warning! W1027: file xmp_static.lib(player.c): redefinition of xmp_end_player ignored
Warning! W1027: file xmp_static.lib(player.c): redefinition of xmp_get_module_info ignored
Warning! W1027: file xmp_static.lib(player.c): redefinition of xmp_get_frame_info ignored
Warning! W1171: locally defined symbol xmp_release_module imported
Warning! W1171: locally defined symbol xmp_end_player imported
```
Not sure what the problem is here.

**Pull request merged**